packages/botonic-nlu/src/ner.js

```diff
@@ -0,0 +1,39 @@
+import { default as nlp } from 'compromise'
+import { default as compromisePlugin } from 'compromise-plugin'
+import { DEFAULT_ENTITIES } from './constants'
+
+function listContainEntity(obj, list) {
```

Comment:
listContainEntity -> listContainsEntity

maybe change arg name from obj to entity?

Reply:
renamed!